src/local/index.js

```diff
@@ -323,6 +349,10 @@ class Local implements Side {
 
   // Delete a file or folder from the local filesystem
   destroy (doc, callback) {
+    if (inRemoteTrash(doc)) {
+      callback(null)
+      return
+    }
```

Comment:
> Makes sense, I'll fix it.

Well, I'm not so sure in the end: `inRemoteTrash(doc)` actually checks whether `doc.path` starts with `.cozy_trash`. So removing the test would actually mean trying to delete some file in `~/Cozy/.cozy_trash/...`, while `~/Cozy/.cozy_trash` is supposed to never exist...